docs/cli.md

```diff
@@ -63,6 +63,7 @@ We use a plugin architecture so that you can install only the dependencies you a
 | [athena](../metadata-ingestion/source_docs/athena.md)                               | `pip install 'acryl-datahub[athena]'`                      | AWS Athena source                   |
 | [bigquery](../metadata-ingestion/source_docs/bigquery.md)                           | `pip install 'acryl-datahub[bigquery]'`                    | BigQuery source                     |
 | [bigquery-usage](../metadata-ingestion/source_docs/bigquery.md)                     | `pip install 'acryl-datahub[bigquery-usage]'`              | BigQuery usage statistics source    |
+| [datahub-file-lineage](../metadata-ingestion/source_docs/file_lineage.md)           | _no additional dependencies_                               | Lineage File source                 |
```

Comment:
suggest calling this `datahub-lineage-file` instead of `datahub-file-lineage`, that way people won't confuse it with lineage between files.

Reply:
Sounds good.